frontend/src/components/App/Layout.tsx

```diff
@@ -202,12 +202,27 @@ export default function Layout({}: LayoutProps) {
       >
         {t('Skip to main content')}
       </Link>
-      <Box sx={{ display: 'flex', [theme.breakpoints.down('sm')]: { display: 'block' } }}>
+      <Box
+        sx={{
+          display: 'flex',
+          overflow: 'auto',
+          paddingTop: { sm: '64px', xs: '72px' },
+          height: '100vh',
+        }}
+      >
         <VersionDialog />
         <CssBaseline enableColorScheme />
         <TopBar />
         <Sidebar />
-        <Main id="main" sx={{ flexGrow: 1, marginLeft: 'initial' }}>
+        <Main
+          id="main"
+          sx={{
+            flexGrow: 1,
+            marginLeft: 'initial',
+            overflow: 'auto',
+            [theme.breakpoints.down('sm')]: { marginTop: { xs: '-12px', sm: '-8px' } },
```

Comment:
that's a lot of new magic values (64px, 72px, -12px, -8px). maybe we can do this change without introducing them? maybe making topbar not positioned absolute or sticky. 
or if that's not possible at least introduce some constants and plan for a possibility that topbar size might be changed in the future

Reply:
if the topbar isn't set to fixed then that defeats the purpose of it right? I think the new values are a good workaround (temporary if we can fix the topbar somehow) so I can abstract them into constants for now